### DISCUSSION TITLE
Text size in markdown field?

### DISCUSSION THREAD

**Discussion by diedder** [Budibase/budibase#6408] [Help] [answered]
Hi,

how can I change the text (standard) size in the markdown-viewer? I couln't find anything here or in the docs...

diedder

**Comment by melohagan** [comment, marked as answer]
Hi @diedder 

You can click on *Edit custom CSS* and make changes to the font size there, e.g. `font-size: large;` or any other valid font size CSS.

**Comment by diedder** [comment]
Thank you @melohagan as always very quick and helpful!
